pipeline/scripts/3-averageTechReplicates.R

```diff
@@ -122,3 +124,61 @@ repl.pattern.filtered <- retVal$pattern
 save(repl.pattern.filtered, file=paste(outdir, "repl.pattern.negative.RData", sep="/"))
 write.table(remove_neg, file=paste(outdir, "miss_infusions_neg.txt", sep="/"), row.names=FALSE, col.names=FALSE ,sep= "\t")
 
+# New: generate TIC plots
+run_name <- basename(outdir)
+TIC_input_dir <- paste(outdir, "2-pklist", sep = "/")
```

Comment:
Try to avoid capitals in your variable names. This is also more consistent with the rest of the code.

Reply:
Capital TIC replaced with tic for variable names.